src/main/java/com/shippo/ExampleBatchShipment.java

```diff
@@ -0,0 +1,214 @@
+		parcel2Map.put("weight", "15");
+		parcel2Map.put("mass_unit", "oz");
+		List<Map<String, Object>> parcels2 = new ArrayList<Map<String, Object>>();
+		parcels2.add(parcel2Map);
+
+
+		// 2nd shipment
+		Map<String, Object> secondShipmentMap = new HashMap<String, Object>();
+		secondShipmentMap.put("address_to", toAddressMap);
+		secondShipmentMap.put("address_from", fromAddressMap);
+		secondShipmentMap.put("parcels", parcels2);
+		Shipment shipment2 = Shipment.create(secondShipmentMap);
+
+		if (shipment2.getStatus().equals("SUCCESS")) {
+			System.out.println(String.format("Shipment created with id: %s", shipment2.getObjectId()));
+		} else {
+			System.out.println(String.format("An Error has occured while creating the 2nd shipment. Messages : %s", shipment2.getMessages()));
+			System.exit(0);
+		}
+
+
+		//Example of retrieving a batch object to check its validation status
+		//For complete reference to the retrieve endpoint: https://goshippo.com/docs/reference#batches-retrieve
+		//This method of polling the batch validation status is for demo purposes only
+		//In practice it is advised to use the batch-create webhook in the user api dashboard: https://app.goshippo.com/api/
+		int maxTimeout = 10;
+		int counter = 0;
+		while (counter < maxTimeout) {
+			Batch retrievedBatch = Batch.get(batch.getId(), 0, null);
+			if (retrievedBatch.getStatus().equals(BatchStatus.VALID)) {
+				break;
+			} else {
+				counter++;
+				try {
+					TimeUnit.SECONDS.sleep(1);
+				} catch (InterruptedException e) {
+					e.printStackTrace();
+				}
+			}
+		}
+		Batch retrievedBatch2 = Batch.get(batch.getId(), 0, null);
+		if (retrievedBatch2.getStatus().equals(BatchStatus.VALID)) {
+			System.out.println(String.format("Batch object %s has status %s ", retrievedBatch2.getId(), retrievedBatch2.getStatus()));
+		} else {
+			System.out.println(String.format("An Error has occured validating the batch."));
+			System.exit(0);
+		}
+
+		// example adding a shipment to batch
+		String[] shipmentsToAdd = { shipment2.getObjectId() };
+		Batch addedShipment = Batch.addShipments(batch.getId(), shipmentsToAdd);
+		if (addedShipment.getStatus().equals(BatchStatus.VALID)) {
+			System.out.println(String.format("Batch now contains shipments : %s ", addedShipment.getBatchShipments().toString()));
+		} else {
+			System.out.println(String.format("An Error has occured adding a shipment."));
+			System.exit(0);
+		}
+
+		// example removing a shipment from batch
+		BatchShipmentCollection batchCollection = addedShipment.getBatchShipments();
+		BatchShipment[] batchShipment = batchCollection.getShipments();
+		String[] shipmentsToRemove = { batchShipment[0].getId().toString() };
+		Batch removedShipment = Batch.removeShipments(batch.getId(), shipmentsToRemove);
+		if (removedShipment.getStatus().equals(BatchStatus.VALID)) {
+			System.out.println(String.format("Batch now contains shipments : %s ", removedShipment.getBatchShipments().toString()));
```

Comment:
Is this logline accurate? Shouldn't it say something like `"Removed shipment : %s "` ?

Reply:
This is the remaining shipments on the batch after the removal